web/themes/custom/urbanobserver/components/03-organisms/site/site-header/site-header.twig

```diff
@@ -5,12 +5,12 @@
 
 <header {{ bem(header__base_class) }}>
   <div {{ bem('inner', [], header__base_class) }}>
-    <div {{ bem('primary', [], header__base_class) }}>
-      <div {{ bem('branding', [], header__base_class) }}>
-        {% block header__branding %}
-          {% include "@organisms/site/site-header/_site-header-branding.twig" %}
-        {% endblock %}
-      </div>
+{#    <div {{ bem('primary', [], header__base_class) }}>#}
+{#      <div {{ bem('branding', [], header__base_class) }}>#}
+{#        {% block header__branding %}#}
+{#          {% include "@organisms/site/site-header/_site-header-branding.twig" %}#}
+{#        {% endblock %}#}
+{#      </div>#}
```

Comment:
Generally commented code is not accepted in the repository unless there is a reason to keep it.
But in our case, whatever